packages/block-library/src/button/editor.scss

```diff
@@ -18,10 +18,15 @@
 		cursor: text;
 		line-height: 1;
 	}
+
+	&:not(.has-text-color) .editor-rich-text__tinymce[data-is-placeholder-visible="true"] + .editor-rich-text__tinymce {
```

Comment:
That’s quite the selector! 😅

Any chance there’s anything less intense we can use? No worries if not.

Reply:
Not really, sadly — the way we create placeholder text in fields like these is rather exotic, so it has to use the plus selector. We also want to make sure that the actual text color is only applied when a user specified one isn't, in that case we want to rely on opacity for the user customized placeholder text color :)